tests/schema/tests.py

```diff
@@ -1673,3 +1673,19 @@ def test_alter_field_default_dropped(self):
             )
             if connection.features.can_introspect_default:
                 self.assertIsNone(field.default)
+
+    def test_add_textfield_unhashable_default(self):
+        # Create the table
+        with connection.schema_editor() as editor:
+            editor.create_model(Author)
+        # Create a row
+        Author.objects.create(name='Anonymous1')
+        # Create a field that has an unhashable default
+        new_field = TextField(default={})
+        new_field.set_attributes_from_name("info")
+        # Fail if an exception is raised
```

Comment:
I don't think try/except/fail is a good pattern. See 071801ccff970682a799ce754431a3c3ce3d6902 for the reasoning.